sklearn/datasets/base.py

```diff
@@ -255,6 +255,12 @@ def load_iris():
 
     Read more in the :ref:`User Guide <datasets>`.
 
+    Parameters
+    ----------
+    return_X_y : boolean, default=False.
+        If true, you will get (X, y), else you will get Bunch.
+
+
     Returns
     -------
     data : Bunch
```

Comment:
You're welcome. Thanks for contributing to scikit-learn.

On Wed, Apr 27, 2016 at 2:58 AM, hongguangguo <notifications@github.com>
wrote:

> In sklearn/datasets/base.py
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_scikit-2Dlearn_scikit-2Dlearn_pull_6704-23discussion-5Fr61217859&d=CwMCaQ&c=8hUWFZcy2Z-Za5rBPlktOQ&r=IgVP_sCIQHhgRgxQoo7gufqSQOvlFIckTGgYIpXBa_4&m=_Nz1aN8wUvItYUqN7zZ8y8zkC1y9ij8rKI4uhID9eZ8&s=Jse05e4p-zfOLNk9rKDXV6bNKkSVXffmV_W6Bp4wlKw&e=>
> :
>
> > @@ -255,6 +255,12 @@ def load_iris():
> >
> >      Read more in the :ref:`User Guide <datasets>`.
> >
> > +    Parameters
> > +    ----------
> > +    return_X_y : boolean, default=False.
> > +        If true, you will get (X, y), else you will get Bunch.
> > +
> > +
> >      Returns
> >      -------
> >      data : Bunch
>
> thanks
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_scikit-2Dlearn_scikit-2Dlearn_pull_6704_files_9feba9d49035952c8a7abeebb44dc9fd403b9f62-23r61217859&d=CwMCaQ&c=8hUWFZcy2Z-Za5rBPlktOQ&r=IgVP_sCIQHhgRgxQoo7gufqSQOvlFIckTGgYIpXBa_4&m=_Nz1aN8wUvItYUqN7zZ8y8zkC1y9ij8rKI4uhID9eZ8&s=tUjKcbmo6aluoBHEmXPwVa0vV4fVH36igpKFHEKUfGY&e=>
>



-- 



*Henry Lin, Research AssistantM.S. Computer ScienceUniversity of Illinois
at Urbana-Champaign 2016847-769-8729 | halin2@illinois.edu
<halin2@illinois.edu>*